tool/teleport/common/teleport.go

```diff
@@ -235,6 +235,8 @@ func Run(options Options) (app *kingpin.Application, executedCommand string, con
 		StringsVar(&dbConfigCreateFlags.AuthServersAddr)
 	dbConfigureCreate.Flag("token", "Invitation token to register with an auth server [none].").Default("/tmp/token").StringVar(&dbConfigCreateFlags.AuthToken)
 	dbConfigureCreate.Flag("rds-discovery", "List of AWS regions in which the agent will discover RDS/Aurora instances.").StringsVar(&dbConfigCreateFlags.RDSDiscoveryRegions)
+	dbConfigureCreate.Flag("azure-mysql-discovery", "List of Azure regions in which the agent will discover MySQL servers.").StringsVar(&dbConfigCreateFlags.AzureMySQLDiscoveryRegions)
```

Comment:
nit: Shouldn't we add Azure-specific options after AWS?

Reply:
Oh, I didn't realize that flag creation order even mattered for our cli parser, good point! I'll move it below AWS stuff